src/theme/globalStyles.ts

```diff
@@ -22,20 +22,23 @@ const GlobalStyle = createGlobalStyle`
 
   h1 {
     color: ${colors.secondaryFontColor};
+    font-family: ${fonts.montserrat};
```

Comment:
We could create some shared blocks here so we don't have to repeat the same styles for all of the h's. 
Like:
```
h1, h2, h3, h5, h6, button {
    font-family: ${fonts.montserrat};
}
```